low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -339,6 +356,41 @@ class SSPLDevDeploy:
         Conf.save(sspl_config_index)
         Conf.save(global_config_index)
 
+    def setup_message_broker(self):
+        from cortx.utils.conf_store import Conf
+        Conf.load("message_bus", "json:///etc/cortx/message_bus.conf")
```

Comment:
dont hard code, this should come come from common config if everyone going to need this, foundation team should make it available and we should just read it from confstore.

Reply:
cortx-utils expects message bus config file at [this ](https://github.com/Seagate/cortx-utils/blob/main/py-utils/src/utils/message_bus/message_bus.py#L27)location. They have not made is configurable. So its clear they expect file at this location.

Reply:
I understand the need, as there is no other way today beyond hardcoding

Reply:
I have removed this, as no longer needed.